org.mixedrealitytoolkit.input/Tests/Runtime/BasicInputTests.cs

```diff
@@ -638,7 +639,7 @@ public IEnumerator HandModelHasRequiredFieldsAndAccessors()
         }
 
         // Returns true iff any of the ProximityDetectors in the scene are currently triggered.
```

Comment:
While you're here, please fix the old typo in the comment.  "iff"

Reply:
Thanks for catching that.  Not a typo though, 'iff' is the abbreviated version of 'if and only if', although expanding it improves readability.  I expanded all 'iff' abbreviations in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/758/commits/c4270ed3e8eefaeee97dbb9e234da7a24c3dec24 , resolving.